src/components/pages/devices/flyouts/deviceDetails/deviceDetails.js

```diff
@@ -391,6 +446,28 @@ export class DeviceDetails extends Component {
 
                 </Section.Content>
               </Section.Container>
+
+              <Section.Container>
+                <Section.Header>{t('devices.flyouts.details.modules.title')}</Section.Header>
+                <Section.Content>
+                  <SectionDesc>
+                    {t("devices.flyouts.details.modules.description")}
+                  </SectionDesc>
+                  <div className="device-details-deployment-contentbox">
+                    {
+                      !moduleQuerySuccessful &&
+                      t('devices.flyouts.details.modules.noneExist')
+                    }
+                    {
+                      moduleQuerySuccessful &&
+                      <div>
```

Comment:
Use componentArray instead of vanilla div

Reply:
done